Modules/TOF/src/PostProcessDiagnosticPerCrate.cxx

```diff
@@ -30,14 +30,26 @@ using namespace o2::quality_control::postprocessing;
 namespace o2::quality_control_modules::tof
 {
 
-const std::string PostProcessDiagnosticPerCrate::mCCDBPath = "TOF/MO/TaskRaw/";
 const int PostProcessDiagnosticPerCrate::mNWords = 32;
 const int PostProcessDiagnosticPerCrate::mNSlots = 14;
 
 PostProcessDiagnosticPerCrate::~PostProcessDiagnosticPerCrate()
 {
 }
 
+void PostProcessDiagnosticPerCrate::configure(const boost::property_tree::ptree& config)
+{
+  const std::string baseJsonPath = "qc.postprocessing." + getID() + ".customization.";
+  mCCDBPath = config.get<std::string>(baseJsonPath + "CCDBPath", "TOF/MO/TaskRaw/");
```

Comment:
Why use a dedicated key instead of `qc.config.conditionDB.url` ? Can yours be different than the main one?

Reply:
OK, I think I misunderstood what the variable stands for in your case...